src/source/geojson_source.js

```diff
@@ -2,8 +2,8 @@
 
 const Evented = require('../util/evented');
 const util = require('../util/util');
-const window = require('../util/window');
 const EXTENT = require('../data/extent');
+const resolveURL = require('../util/mapbox').resolveURL;
```

Comment:
@jfirebaugh `util.js` already depends on `window`, so putting `resolveURL` in `util.js` creates a cyclic dependency. `resolveURL` could be moved to `ajax.js`, or left in `geojson_source.js` which was its only usage anyways